airbyte-cdk/python/file_based_build_customization.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from __future__ import annotations
+
+import textwrap
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from dagger import Container
+
+
+def setup_nltk(connector_container: Container) -> Container:
+    """
+    Seeds the connector with nltk data at build time. This is because the nltk data
+    is large and takes a long time to download. It runs a python script that downloads
+    the data following connector installation.
+    """
+
+    nltk_python_script = textwrap.dedent(
+        """
+        import nltk
+        nltk.download('punkt')
+        nltk.download('averaged_perceptron_tagger')
+        """
+    )
+    connector_container = (
+        connector_container.with_new_file("/tmp/nltk_python_script.py", nltk_python_script)
+        .with_exec(["python", "/tmp/nltk_python_script.py"], skip_entrypoint=True)
+        .with_exec(["rm", "/tmp/nltk_python_script.py"], skip_entrypoint=True)
+    )
```

Comment:
Could you comment on which system path the nltk data will be written?

Reply:
Done